### DISCUSSION TITLE
Backups

### DISCUSSION THREAD

**Discussion by AdamSnow** [Freika/dawarich#187] [Q&A] [answered]
How is everyone backing up their database?

**Comment by Freika** [comment]
I personally just hit "Export" button every now and then, but I think I'll write a guide on how to properly back up the database at some point. Meanwhile, Immich has a good approach described in their docs: https://immich.app/docs/administration/backup-and-restore

**Comment by Freika** [comment, marked as answer]
I added a page on backup and restore, ripped it off Immich a bit 🌚 

https://dawarich.app/docs/tutorials/backup-and-restore
